src/build/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Utilities to build a Bevy app targeting either native or web platforms.
```

Comment:
nit: Maybe "utilities" is misleading here, this file basically defines the entire logic of the command